roles/splunk_search_head/tasks/search_head_clustering.yml

```diff
@@ -75,16 +75,3 @@
   retries: "{{ shc_sync_retry_num }}"
   delay: "{{ retry_delay }}"
   no_log: "{{ hide_password }}"
-
-- name: Destructive sync search head
```

Comment:
should we instead make it optional? would it cause any issues while performing upgrades?

Reply:
This is one of those tasks that took almost 5 minutes to complete. According to this doc https://docs.splunk.com/Documentation/Splunk/9.2.0/DistSearch/HowconfrepoworksinSHC#Why_a_recovering_member_might_need_to_resync_manually we shouldn't need to perform a destructive sync unless the cluster is recovering a member, and then it's only necessary on that one instance. We always could leave this off by default and have some flag to run it if possible.